expressEngine/user/controller/user.js

```diff
@@ -1,15 +1,20 @@
 // import base controller
-import { BaseController } from '../../lib/controller/index';
+import { BaseController } from '../../lib/controller';
 
 // import service
-import { UserService } from '../service/index';
+import { UserService } from '../service';
 
 class Controller extends BaseController {
   create = async (data) => {
     const userData = await UserService.registerUser(data);
 
     return { userId: userData.userId };
   };
+
+  delete = async (data) => {
+    const { id } = data.params;
+    await UserService.deleteUser(id);
```

Comment:
@adilsuccessive Instead of deleteUser, use delete.

Reply:
Done with the change